tools/buildsteps/windows/make-mingwlibs.bat

```diff
@@ -5,8 +5,9 @@ PUSHD %~dp0\..\..\..
 SET WORKDIR=%CD%
 POPD
 
-REM recreates ffmpeg build dir in case it does not exist
+REM recreates clean ffmpeg build dir
 SET BUILD_DIR=%WORKDIR%\project\BuildDependencies\build
+IF EXIST %BUILD_DIR% rmdir %BUILD_DIR% /S /Q
```

Comment:
Did you need this change?

Workflow for jenkins is to run the prepare-env.bat script first, which does this (as part of this PR). So having this in here is only relevant for regular dev's, did you want to wipe the folder out always for that use case?

Reply:
Yes, because regular dev's usually want build ffmpeg when version changes and if build dir is not empty may cause problems also even rebuild same version may cause issues i.e. patches not apply, etc.